metaflow/flowspec.py

```diff
@@ -519,6 +519,7 @@ def _get_foreach_item_value(self, item: Any):
         str
             The value to use for the item.
         """
+        MAXIMUM_VALUE_CHARACTERS = 30
```

Comment:
As discussed, I would move this to config and keep the default to 30 but internally we can set to 25. Apart from that, LGTM.

Reply:
Added the config.